lib/fs.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const metasync = require('metasync');
+
+const error = (err, options, cb) => {
+  if (options.errorLog) {
+    options.errorLog.write(err.stack);
+    cb();
```

Comment:
IFAIU, the error should be always passed to a callback and duplicated to an `options.errorLog` if it is specified.

Reply:
Like this?
```javascript
  if (options.errorLog) {
    options.errorLog.write(err.stack);
  } 
  cb(err);
```

Reply:
Yes, though you can make an if statement one-liner.
```js
  if (options.errorLog) options.errorLog.write(err.stack);
  cb(err);
```